packages/strapi-admin/package.json

```diff
@@ -125,5 +126,8 @@
     "npm": ">=6.0.0"
   },
   "license": "SEE LICENSE IN LICENSE",
-  "gitHead": "231263a3535658bab1e9492c6aaaed8692d62a53"
+  "gitHead": "231263a3535658bab1e9492c6aaaed8692d62a53",
```

Comment:
❓ I'm not used to this attributes, what does it aim for?

Reply:
It is used for npm publish. Lerna creates it temporarily on publish & should remove it after release in theory :) 

Haven't investigated why it doesn't reset it yet